CONTRIBUTING.md

```diff
@@ -54,9 +54,15 @@ We also follow these project-specific guidelines:
   constructors to all AutoValue classes to prevent classes in other packages
   from extending them.
 
+## Building instrumentation-java
+
+Run `./gradlew clean assemble check` on OS X or Linux, or run
```

Comment:
AFAICT they do exactly the same actions (I ran both, compared output). But there may be some difference in the future (?) so probably better to keep the `assemble check` explicit.